geomstats/geometry/base.py

```diff
@@ -555,3 +555,97 @@ def projection(self, point):
         projected : array-like, shape=[..., dim]
             Projected point.
         """
+
+
+class ComplexOpenSet(ComplexManifold, abc.ABC):
```

Comment:
If everything is equal to `OpenSet`, should we really create `ComplexOpenSet`?

If the answer is yes, it is better to repeat code here than call to `OpenSet`.

Reply:
Yes, I need `ComplexOpenSet` as it inherits from `ComplexManifold` whereas `OpenSet` inherits from `Manifold`. 
I had some failing tests using `OpenSet`.
Ok, I will repeat code.

Reply:
Done.